gradle.properties

```diff
@@ -1 +1,2 @@
-org.gradle.jvmargs=-Xmx2048m -Xms512m -XX:MaxPermSize=768m
+org.gradle.jvmargs=-Xmx1024m -Xms512m -XX:MaxPermSize=768m
```

Comment:
Please exclude modifications to this file from the commit. This is needed for CI builds.

Reply:
sry, first android studio project and forgot to change this back.

Reply:
I forgot to tell you about the captures folder which mistakenly entered the commit too.